README.md

```diff
@@ -11,6 +11,7 @@ This is the self-serve portal for UID2 participants. It enables a range of opera
 - VS Code (or equivalent). Note that if you don't use VS Code, you will need to find equivalent extensions for things like linting.
 
 Recommended VS Code extensions:
+
```

Comment:
In IntrelliJ the markdown rendering of table below breaks with no newline, adding it fixes it.